_posts/2023-06-15-user-survey-2023.md

```diff
@@ -0,0 +1,240 @@
+With ASIC designers being traditionally more risk-averse (for good reasons), this increase can be seen as indication that cocotb is being trusted to deliver good verification outcomes.
+
+<div class="row p-5">
+  <div class="col">
+    <img src="/assets/blog/2023-06-15-user-survey-2023/3-target.svg" class="img-fluid mx-auto d-block" alt="Target platform">
+  </div>
+  <div class="col">
+    <img src="/assets/blog/2023-06-15-user-survey-2023/4-designlanguages.svg" class="img-fluid mx-auto d-block" alt="design languages">
+  </div>
+</div>
+
+
+## Why not something else?
+
+**SystemVerilog UVM remains the primary verification choice -- but cocotb users don't really like it.**
+
+cocotb isn't the only verification approach out there, and (we have to admit) not the most common one either.
+SystemVerilog UVM has been the go-to choice for a while, and that's reflected in our user base as well.
+Roughly one third of users use formal verification, plain VHDL testbenches, or other Python-based verification approaches.
+C or C++-based verification approaches (often combined with UVM) are used by 23 percent of our respondents.
+
+The answers leave a lot of room for follow-up questions, like "how much of a design is actually verified using formal?" or "what does the in-house Python testbench look like?".
+We'd certainly be interested in an answer -- reach out over social media to tell us more!
+
+<img src="/assets/blog/2023-06-15-user-survey-2023/6-otherapproaches.svg" class="img-fluid mx-auto d-block p-5" alt="design languages">
+
+When asked about the pain points of these other verification approaches, cocotb users repeatedly mentioned
+* the complexity (and learning curve) of UVM,
+* poor CI/test framework integration, and
+* the amount of boilerplate code and effort required until the actual writing of the test can start.
+
+## How our users run cocotb
+
+**The overwhelming majority of cocotb users uses a recent version of cocotb on Linux.**
+
+Cocotb is open source, and freedom of choice and diversity has long been one of the key properties in open source.
+Still, knowing the platforms the majority of our users run on is essential to properly focus our engineering and testing efforts.
+Thankfully, the results we got match what we were expecting:
+
+The overwhelming majority of our users runs Linux, typically Red Hat Enterprise Linux or one of its derivatives, which tends to be most common in industry settings, or Debian or Ubuntu, which is used widely in "less managed" settings, as well as in CI.
+
+Even though the cocotb project has spent a very significant amount of effort in engineering and support on it, our Windows user base remains comparably small.
+
+<div class="col">
+  <img src="/assets/blog/2023-06-15-user-survey-2023/8-os-distro.svg" class="img-fluid mx-auto d-block" alt="operating system or Linux distribution">
```

Comment:
Maybe patch this file to replace "Redhat Enterprise Linux" with "RHEL" so that the version numbers are not cut off.

Reply:
:+1: Good point, updated.